### PR TITLE
Test dependancies

### PR DESCRIPTION
**Pull request by ivastar**
This pull request:
- Adds the 3 remaining pip-installable dependencies
- Fixes the symlinking of the template directory

### CODE REVIEW

File: grizli/utils.py [hunk 1]
```diff
@@ -6005,24 +6005,20 @@ def symlink_templates(force=False):
     #     return False
 
     module_path = os.path.dirname(__file__)
+    templates_path = os.path.join(module_path, 'data/templates')
```
Comment (jdavies-st) [inline grizli/utils.py:6007]: Preferable:
```
    templates_path = os.path.join(module_path, 'data', 'templates')
```

**Review by jdavies-st — APPROVED** [review]
Looks good.  Minor comment below.

File: grizli/utils.py (hunk 1 above)
Comment (ivastar) [reply grizli/utils.py:6007]: @jdavies-st can you do this as a suggestion so I can just approve it?

Comment (jdavies-st) [inline grizli/utils.py:6007]:
```suggestion
    templates_path = os.path.join(module_path, 'data', 'templates')
```

**Pull request merged**